_rules/error-field-is-not-identified-using-color-only-i3z4v8.md

```diff
@@ -0,0 +1,166 @@
+---
+id: i3z4v8
+name: Error field is not identified using color differences only
+rule_type: atomic
+description: |
+  This rule checks that error fields can be identified not only by color differences but through another visual means.
+accessibility_requirements:
+  wcag21:1.4.1: # Use of Color (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  -  DOM Tree
+  -  CSS Styling
+acknowledgments:
+  authors:
+    -  Giacomo Petri
+---
+
+## Description
+
+This rule checks that errors are not identified only through color differences.
+
+## Applicability
+
+This rule applies to any [visible][] [input error][] together with:
```

Comment:
Not fully convinced that "input error" is unambiguous enough.
> information provided by the user that is not accepted

Does that mean that the field in which it was inputed is put in a validation error state? Or just that some code checked it and rejected it, without changing the validation state? Actually, this refers to the information, not even to the form field, is it what you want the rule to target?